src/test/suite/fixtures/recorded/headTail/takeHead.yml

```diff
@@ -0,0 +1,31 @@
+spokenForm: take head
+languageId: typescript
+command:
+  actionName: setSelection
+  partialTargets:
+    - type: primitive
+      modifier: {type: head}
+  extraArgs: []
+marks: {}
+initialState:
+  documentContents: |
+
+    const value = "Hello world";
+
+    const value = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 9}
+      active: {line: 1, character: 9}
+finalState:
+  documentContents: |
+
+    const value = "Hello world";
+
+    const value = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 9}
+      active: {line: 1, character: 0}
```

Comment:
Do we want these reversed? I can see the logic, but I'd imagine it's more common to want it forward, so would prob make sense to have that be the default

Reply:
I really like them like they are to be honest. Its define as past start of line after all.

Reply:
And to be real these modifers are mostly usefull for chuck or clear. With a take it doesn't really matter where the active is because its gonna get overewritten if you insert anything.

Reply:
Fair enough; if they're working well for you this way and it's simpler let's leave them!